src/transformers/modeling_tf_bert.py

```diff
@@ -88,124 +92,91 @@
 ]
 
 
-class TFBertEmbeddings(tf.keras.layers.Layer):
-    """Construct the embeddings from word, position and token_type embeddings."""
+class TFBertPreTrainingLoss:
+    """
+    Loss function suitable for BERT-like pre-training, that is, the task of pretraining a language model by combining
+    NSP + MLM.
```

Comment:
```suggestion
    Next Sentence Prediction and Masked Language Modeling.
```
We have plenty of space in the docs, no need for abbreviations :-)